_inc/client/components/product-expiration/index.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { moment, translate as __, localize } from 'i18n-calypso';
```

Comment:
We don't, I guess I wasn't sure which is better here :)

Reply:
It doesn't really matter for Jetpack, and mostly we use just `translate`. 😉 In Calypso, `localize` is preferred.

Reply:
Got it! How does centralizing the component affect things?

Reply:
Since we can use `i18n-calypso` into the centralized components, I'd suggest using `localize()` when you need a HoC and the `useTranslate` hook when you have a functional component. These are already used in packages so we won't set a precedent.

Reply:
Although I'm not sure we want to limit consumers to using i18n-calypso for i18n.